src/dev/flang/tools/docs/Util.java

```diff
@@ -155,17 +155,13 @@ static enum Kind {
 
     static Kind classify(AbstractFeature af) {
```

Comment:
Is this still used anywhere?

Reply:
Yes for the navigation that can be accessed with the hamburger menu. That might need restructuring as well, but I'm not sure what the exact purpose of that is. I will create a separate issue for that.